src/library/scala/collection/immutable/ListSet.scala

```diff
@@ -13,84 +13,74 @@ package immutable
 import generic._
 import scala.annotation.tailrec
 
-/** $factoryInfo
- *  @define Coll immutable.ListSet
- *  @define coll immutable list set
- *  @since 1
- */
+/**
+  * $factoryInfo
+  *
+  * Note that each element insertion takes O(n) time, which means that creating a list set with
+  * n elements will take O(n^2^) time. This makes the builder suitable only for a small number of
```

Comment:
ah, in that case you could use the same up here: https://github.com/scala/scala/pull/5103/files/78f1fdd0646e6da5de3f86eaec265d9b5d71b1f4..36a46b1cc2734504c95881c20b2495891c7b9bd7#diff-9f3860217088c43b2414a59d6ce4b2acR20